jenkins-design-language/src/js/components/material-ui/SvgIcon/SvgIcon.js

```diff
@@ -90,6 +78,7 @@ class SvgIcon extends Component {
         {...other}
         onMouseEnter={this.handleMouseEnter}
         onMouseLeave={this.handleMouseLeave}
+        fill={color ? color : 'white'}
```

Comment:
is this necessary?

Reply:
that fill attr is kind if necessary because white was the default color in the react-material-icons, if I remove that, a lot of icons become black and we need to write css to make them white again, I think white is a good default color because in a lot of places that we use the icons they are white, there is no place where they are black so I would leave this for convenience

Reply:
Can't we just make a low-specificity CSS rule that defaults them to white? e.g. add a `className="icon"` to `SvgIcon` and then use `svg.icon { fill: #fff }` ?

Reply:
yup, will do that, I will add the other default css icon styles in that class as well